rosdep/base.yaml

```diff
@@ -3398,9 +3398,11 @@ npm:
   fedora: [npm]
   ubuntu: [npm]
 ntp:
+  gentoo: [net-misc/ntp]
```

Comment:
Is is possible to add [debian](https://packages.debian.org/search?keywords=ntp&searchon=names&suite=all&section=all) and [fedora](https://admin.fedoraproject.org/pkgdb/package/rpms/ntp/) here as well ?